src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,19 @@
 
 public class MainApp {
 
```

Comment:
Please remove the redundant empty line after the class declaration to adhere to the coding standards mentioned in the checklist.